tests/panoramic/cli/local/test_reader.py

```diff
@@ -45,6 +53,18 @@ def test_file_package_read_models(tmp_path):
     with model_file.open('w') as f:
         f.write('model_name: test-model')
 
-    package = FilePackage(name='dataset1', data_source_file=Mock(), model_files=[model_file])
+    package = FilePackage(name='dataset1', data_source_file=Mock(), model_files=[model_file], field_files=[])
 
     assert list(package.read_models()) == [({'model_name': 'test-model'}, model_file)]
+
+
+def test_file_package_read_fields(tmp_path):
+    field_file = Paths.fields_dir(tmp_path) / f'test_field{FileExtension.FIELD_YAML.value}'
+    os.makedirs(os.path.dirname(field_file), exist_ok=True)
+
+    with field_file.open('w') as f:
+        f.write('slug: field_slug')
+
+    package = FilePackage(name='dataset1', data_source_file=Mock(), model_files=[], field_files=[field_file])
+
+    assert list(package.read_fields()) == [({'slug': 'field_slug'}, field_file)]
```

Comment:
Can you add tests for `get_global_fields` ?

Reply:
Should be covered by validation tests added here https://github.com/panoramichq/panoramic-cli/pull/12